docs/IE_DG/supported_plugins/HETERO.md

```diff
@@ -66,6 +66,8 @@ auto network = core.ReadNetwork("Model.xml");
 auto executable_network = core.LoadNetwork(network, "HETERO:FPGA,CPU");
 ```
 
+> **NOTE**: `InferenceEngine::Core::QueryNetwork` does not take into account affinities set by a user, it quries layers support based on device capabilities.
```

Comment:
If it doesn't change the meaning, i'd put it like this:
```
> **NOTE**: `InferenceEngine::Core::QueryNetwork` does not depend on affinities set by a user, but queries for layer support based on device capabilities.
```

Reply:
fixed, thanks!